src/components/common/Dialogs/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Id } from '@colony/colony-js';
+import { useNavigate } from 'react-router-dom';
+import { string, object, array, number, boolean, InferType } from 'yup';
+
+import {
+  mergePayload,
+  withKey,
+  mapPayload,
+  pipe,
+  withMeta,
+} from '~utils/actions';
+import { ActionTypes } from '~redux';
+import { WizardDialogType } from '~hooks';
+import Dialog, { ActionDialogProps, DialogProps } from '~shared/Dialog';
+import { ActionHookForm as Form } from '~shared/Fields';
+
+import { getPermissionManagementDialogPayload } from './helpers';
+import PermissionManagementForm from './PermissionManagementForm';
+
+const validationSchema = object()
+  .shape({
+    domainId: number().required(),
+    user: object()
+      .shape({
+        profile: object()
+          .shape({
+            displayName: string(),
+          })
+          .defined()
+          .nullable(),
+        walletAddress: string().address().required(),
+      })
+      .defined()
+      .nullable(),
```

Comment:
Yep, SingleUserPicker stores the whole user object inside the forms where it's used.

We can update that, perhaps it's something we could take into consideration when we rework that component (I remember that we planned to do that in the Dapp but then we moved to the CDapp), or when we do the UI refresh? Otherwise we could just create a issue for this particular change, it shouldn't be too complicated